common/djangoapps/microsite_configuration/backends/base.py

```diff
@@ -312,6 +312,7 @@ def get_template_path(self, relative_path, **kwargs):
         from microsite_configuration.microsite import get_value as microsite_get_value
 
         microsite_template_path = microsite_get_value('template_dir', None)
+        relative_path = template_path[1:] if template_path.startswith('/') else template_path
 
         if not microsite_template_path:
```

Comment:
Just for readability, let's move this block up to line 315 and move relative_path closer to search_path